src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./components/App/container";
+import { fetchConfig } from "./configuration";
+import { Provider } from "react-redux";
+import store from "./store";
+
+fetchConfig().then((config) => {
```

Comment:
That's not gonna work that way.

The server won't let you access the initial state without getting authenticated. We need the webui to be available before we can acutally get the initial config state.

It's not obvious when looking at the previous desktop app because i made the authentication automatic, but in a web browser context we need the user to enter his credentials / server address.

![image](https://user-images.githubusercontent.com/7503585/103276975-3536c200-49c8-11eb-8d0f-cc38b26429ce.png)
![image](https://user-images.githubusercontent.com/7503585/103276987-3cf66680-49c8-11eb-9b03-10dd4499185b.png)

Reply:
The authentication (not that i hink abou that i may have forgot to tell you about that...) is not something to take care right now, but we need to keep it in mind otherwise it will be a pain in the back to implement later

Reply:
Yes of course but that what I said in the slack chat : I will move this to another part in next PR...this is just becaue for this PR I was focused on configuration fetching and then I wanted to immediatly know for any problem even before rendering the DOM